src/main/java/com/nutomic/syncthingandroid/syncthing/SyncthingService.java

```diff
@@ -429,9 +432,19 @@ public IBinder onBind(Intent intent) {
      */
     @Override
     public void onDestroy() {
-        super.onDestroy();
-        Log.i(TAG, "Shutting down service");
-        shutdown();
+//        super.onDestroy();    Call of super not required -> https://developer.android.com/guide/components/services.html
```

Comment:
Well, I made this comment only to inform you why I deleted that line and put a reference for the reason there. The call to super isn't required according to Google. So to follow your style to remove unecessary code, I deleted that line.

Regarding the javadoc:
Shall I describe the change in a javadoc assigned to onDestroy?
Then also a comment to the onPostExecute would be appropriate as the other half of the change lives there.